app/assets/javascripts/katello/hosts/host_and_hostgroup_edit.js

```diff
@@ -134,18 +149,20 @@ KT.hosts.onKatelloHostEditLoad = function(){
     });
 };
 
-KT.hosts.toggle_installation_medium = function() {
-    var lifecycle_environment_id, content_source_id, architecture_id, operatingsystem_id, content_view_id;
+KT.hosts.toggle_installation_medium = function(content_view_id) {
```

Comment:
Due to the way this function is now used above, we might be given a content_view_id, so need to account for that